src/main/java/ua/net/tokar/json/rainbowrest/RainbowRestWebFilter.java

```diff
@@ -188,6 +219,24 @@ private void filterTree(JsonNode tree, Set<String> includedFields ) {
             }
         }
     }
+
+    private void excludeFieldsFromTree( JsonNode tree, Set<String> excludeFields ) {
```

Comment:
i think that both operations could be done in single run. the only modification should be:

````
if ( !includedFields.contains( key ) || excludeFields.contains( key ) ) {
````
instead of
````
if ( !includedFields.contains( key ) ) {
````